R/graph.decay.curves.R

```diff
@@ -19,10 +19,10 @@
 #' * If the user directly specifies the `rows_to_graph` argument, then `n_curves` has no effect.
 #' @examples
 #' library(dplyr) # loads the `%>%` operator and `dplyr::filter()`
-#' "https://osf.io/download/rtw5k/" %>%
-#' load_curve_params() %>%
-#' dplyr::filter(antigen_iso == "HlyE_IgG") %>%
-#' serocalculator:::plot_curve_params_one_ab()
+#'   "https://osf.io/download/rtw5k/" %>%
```

Comment:
line 22 doesn't need an indent, I think; check the formatted help file and make sure it looks correct?

Currently, I'm getting this:

![image](https://github.com/UCD-SERG/serocalculator/assets/2474437/3acd5231-793d-4202-a6f5-470f8784c58b)

The second line is the start of a new command, so it shouldn't be indented

Reply:
Removed indent and updated this to put the URL inside of `load_curve_params()`